File: LeetCode/SortingTheSentence.js
/**
 * @param {string} s
 * @return {string}
 */
 var sortSentence = function(s) {
    
     
    let stringarr=s.split(" ")
    let updatedarr=[]
    for(let i=0;i<stringarr.length;i++)
    {
        updatedarr.push("")
    }
    stringarr.forEach(str=>{
        let index=str.charAt(str.length-1)
        updatedarr[index]=str.slice(0,str.length-1)
    })

    let sentence=""
    updatedarr.forEach(word=>{
        sentence+=word + " "
    })
    return sentence.trim()
    
};

File: LeetCode/SortingTheSentence.py
class Solution:
    def sortSentence(self, s: str) -> str:
        
        arr=s.split(" ")
        print(arr)
        arr2=["" for  i in arr]
        
        for i in arr:
            arr2[int(i[len(i)-1])-1]=i[:len(i)-1]
        return " ".join(arr2)
            